spec/fabricators/space_fabricator.rb

```diff
@@ -2,7 +2,9 @@
 
 Fabricator(:space) do
   title { Faker::Name.first_name }
-  address { Faker::Address.full_address }
+  address { Faker::Address.street_address }
+  post_number { Faker::Address.postcode }
```

Comment:
I've added this in one of my branches too, but I think this will work. One note: Fakers postcodes use the American standard, not the Norwegian one - so numbers are longer. Might cause some issues down the line, so I used a random four digit number instead.

Reply:
Her er min endring: https://github.com/lnu-norge/lokaler.lnu.no/pull/136/files#diff-fedfc07db23064ab5d45a7f6af348ad879d2db04ebb67901dd2fae08904f33ae